lifecycle-operator/controllers/options/keptnconfig_controller.go

```diff
@@ -51,8 +55,52 @@ func NewReconciler(client client.Client, scheme *runtime.Scheme, log logr.Logger
 	}
 }
 
+// variables needed for the Keptn RestAPI
+const restApiPort = 8080
+
+var restApiDeployment = &appsv1.Deployment{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "keptn-rest-api",
```

Comment:
renaming will be necessary since https://github.com/keptn/lifecycle-toolkit/pull/3654/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R24 calls it `keptn-gateway`